### PR TITLE
Update AboutUs with Leo's info

### PR DESCRIPTION
**Pull request by leowolf275**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,55 +5,16 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](https://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e1290028@u.nus.edu`
```
Comment (youngseopark05) [inline docs/AboutUs.md:8]: Maybe we can show our emails in a list, here's what I put down for this line:
```
You can reach us at the following emails:
- `youngseopark@u.nus.edu`
```

File: docs/AboutUs.md [hunk 2]
```diff
@@ -5,55 +5,16 @@ title: About Us
...
 
 ## Project team
 
-### John Doe
+### Leonardo Wolf
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/leowolf275.png" width="200px">
```
Comment (youngseopark05) [inline docs/AboutUs.md:14]: I found that using local paths causes a bug that doesn't let the image render, so maybe we should use the generated asset link: https://ay2425s1-cs2103t-w14-4.github.io/tp/images/leowolf275.png ?

Also maybe we should add alt text for accessibility or see whether there's an issue in the image rendering? Here's an example:
`Youngseo Park's picture`

File: docs/team/leowolf275.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Leonardo Wolf's Project Portfolio Page
```
Comment (youngseopark05) [inline docs/team/leowolf275.md:3]: Maybe we can make the title more personal? For example this is my page's title:
```
---
layout: page
title: 👋 Hi there! I'm Youngseo Park!
---
```

**Comment by youngseopark05** [comment]
Just some nits about the AboutUs page and individual page.

Please feel free to make the quick changes and ping me so I can resolve the merge conflicts and merge.

File: docs/AboutUs.md (hunk 1 above)
Comment (bryanjhc) [reply docs/AboutUs.md:8]: @youngseopark05 I think that's a good idea!

File: docs/AboutUs.md (hunk 2 above)
Comment (leowolf275) [reply docs/AboutUs.md:14]: Thank you!

**Review by youngseopark05 — APPROVED** [review]
LGTM!

File: docs/AboutUs.md (hunk 2 above)
Comment (youngseopark05) [reply docs/AboutUs.md]: Fixed!

File: docs/AboutUs.md (hunk 1 above)
Comment (youngseopark05) [reply docs/AboutUs.md]: Fixed!

**Pull request merged**